plugins/org.python.pydev/src/org/python/pydev/utils/Messages.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2016 by Brainwy Software LTDA. All Rights Reserved.
+ * Licensed under the terms of the Eclipse Public License (EPL).
+ * Please see the license.txt included with this distribution for details.
+ * Any modifications to this file must keep this entire header intact.
+ * 
+ * Copied from the JDT implementation of
+ * <code>org.eclipse.jdt.internal.corext.util.Messages</code>.
+ */
+package org.python.pydev.utils;
+
+import com.ibm.icu.text.MessageFormat;
```

Comment:
I think that we don't need the com.ibm.icu dependency: you should be able to use the standard String.format() for this (code from JDT may be before java 1.5, but we're on java 7 now anyways, so the com.ibm.icu dependency should probably be removed).